cli/command/config/create_test.go

```diff
@@ -105,3 +105,32 @@ func TestConfigCreateWithLabels(t *testing.T) {
 	assert.NoError(t, cmd.Execute())
 	assert.Equal(t, "ID-"+name, strings.TrimSpace(cli.OutBuffer().String()))
 }
+
+func TestConfigCreateWithTemplatingDriver(t *testing.T) {
```

Comment:
Test for config create without driver?

Reply:
There's a test above that doesn't have a driver, is that what you meant?

Reply:
We should check and make sure Templating is nil in that case, I think?

Reply:
This would be done already if the test used `DeepEqual` on the entire spec instead of looking at specific fields. Maybe we should do that here as well.